lib/sales_analyst.rb

```diff
@@ -101,18 +96,6 @@ def total_revenue_by_date(date)
     end
   end
 
-  def get_invoices_for_revenue(date)
-    all_invoices.values.find_all do |invoice|
-      invoice.created_at.to_i == date.to_i
-    end
-  end
-
-  def get_invoice_items_for_revenue(date)
-    get_invoices_for_revenue(date).map do |invoice|
-      @sales_engine.invoice_items.find_all_by_invoice_id(invoice.id)
-    end
-  end
-
```

Comment:
assume this was moved out to sales_enginey things?

Reply:
It was moved from sales_analyst to invoice_analyst module